### ISSUE TITLE
Compile?

### ISSUE THREAD

**Issue by Jopana** [IndirectDarkMatterSearchesIFAE/ObservationOptimization#15]
@Jopana

Perhaps we should create a makefile?

With cmake it is very easy. We could use as an example [flexIRF](https://github.com/cta-observatory/flexIRF).

**Comment by micheledoro** [comment]
Hi Quim, did you investigate this further?

**Comment by Jopana** [comment]
No, I did not. But definitely we should compile the code to speed it up.
